modules/docs/lib/widgets/external.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import {ExternalHyperlink} from '@workday/canvas-kit-react/button';
+
+import {ExternalSymbolValue} from '../../docgen/docTypes';
+
+import {registerWidget} from '../Value';
+
+registerWidget<ExternalSymbolValue>('external', ({value}) => (
+  <ExternalHyperlink href={value.url} style={{fontFamily: 'inherit'}}>
```

Comment:
the inline style here for `fontFamily` seems odd, like I am surprised it's needed, but if you do I trust there's a reason

Reply:
It is subtle:
<img width="146" alt="image" src="https://user-images.githubusercontent.com/338257/221688895-a9dd26ef-e3da-4c06-b522-ae16224cdcae.png">

The top is with `inherit` override. The `Hyperlink` component hard-codes our Canvas font while Storybook uses a different font family and it looks a little weird in Storybook. It will look fine on the Canvas Site. It's one of those things that drive designers nuts.